apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import {Route, Switch} from 'react-router-dom';
+import TeacherDashboardNavigation from './TeacherDashboardNavigation';
+
+export default class TeacherDashboard extends Component {
+  render() {
+    return (
+      <div>
+        <TeacherDashboardNavigation/>
+        <Switch>
+          <Route
+            exact path="/"
+            component={props => <div>Progress content goes here - default for now.</div>}
+          />
```

Comment:
Should we just omit this `Route` and fall through to the default, to avoid implementing it twice?